dxp/liferay_global_event_handler.md

```diff
@@ -0,0 +1,213 @@
+# Liferay’s Global Event Handler
+
+Liferay’s event handler helps the developers for creating/listening/firing/removing custom events that can be listened in whole DXP which is rely on the YUI3 event target implementation. See the implementation [here](https://github.com/liferay/yui3/blob/master/src/event-custom/js/event-target.js#L120).
```

Comment:
I don't think we should emphasize YUI itself, because it is an _implementation detail_ which may well change in the future, even as we maintain the same outward facing API. Still fine to link to it, but I just wouldn't _emphasize_ it so much. In other words instead of:

> rely on the YUI3 event target implementation. See the implementation [here](https://github.com/liferay/yui3/blob/master/src/event-custom/js/event-target.js#L120).

You could just link one of the words in the first sentence:

> [Liferay's global event bus implementation](https://github.com/liferay/yui3/blob/master/src/event-custom/js/event-target.js#L120) allows developers... (etc)

If you wanted to keep your initial wording, you would want to avoid linking to the word "here" anyway (bad for screen readers, because when the user asks for a list of links on the page, "here" doesn't convey any useful context to them); instead of:

> rely on the YUI3 event target implementation. See the implementation [here](https://github.com/liferay/yui3/blob/master/src/event-custom/js/event-target.js#L120).

you would do:

> rely on [the YUI3 event target implementation](https://github.com/liferay/yui3/blob/master/src/event-custom/js/event-target.js#L120).

Overall, maybe some more natural English phrasing for the whole thing would be:

> [Liferay's global event bus implementation](https://github.com/liferay/yui3/blob/master/src/event-custom/js/event-target.js#L120) allows developers to publish and listen to custom events from anywhere inside DXP.

(The bit about creating/removing is probably implicit.)

Reply:
>  Still fine to link to it, but I just wouldn't emphasize it so much.

Makes sense. I forgot to update PR's questions section for asking if we have a ticket for this kinda modernization. I tried now searching for things with `FI_Technical_Debt` label and I didn't found :/

/cc @jbalsas

Reply:
Me and @dsanz will be creating tickets as we build our roadmap and backlog. You knowing this can help us fill in the gaps in the next weeks if you don't see it around.

Reply:
> Liferay's global event bus implementation allows developers to publish and listen to custom events from anywhere inside DXP.

Looks fine :)